### ISSUE TITLE
Confidence score

### ISSUE THREAD

**Issue by joel-simp** [THUDM/CogVLM#496]
Hi, How to check the confidence score or probability score of the generated text? Throwing me error when I try to find the logits
<img width="1039" alt="Screenshot 2024-06-19 at 12 20 19 PM" src="https://github.com/THUDM/CogVLM/assets/170931221/b12058c2-81b6-4c56-b620-32959112b124">

**Comment by Dineshkumar-Anandan-ZS0367** [comment]
Apply normal ligits using torch and you need to pass output_scores=True to the model.gen function
